src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -76,33 +77,49 @@ public String getUrlForWiki(boolean addWWW) {
   }
 
   public String getUrlForWiki(String wikiName, boolean addWWW) {
-    EnvType env = Configuration.getEnvType(this.env);
+    EnvType envType = Configuration.getEnvType(this.env);
     final String wikiaName = getWikiaGlobalName(wikiName);
 
     String www = "";
     if (addWWW) {
       www = "www.";
     }
 
-    switch (env) {
+    switch (envType) {
       case DEV: {
         String devBoxOwner = this.env.split("-")[1];
-        return String.format(DEV_URL_FORMAT, www, wikiaName, devBoxOwner, env.getWikiaDomain());
+        return String.format(DEV_URL_FORMAT, www, wikiaName, devBoxOwner, envType.getWikiaDomain());
       }
       case PROD: {
-        return String.format(PROD_URL_FORMAT, www, wikiaName, env.getWikiaDomain());
+        return String.format(PROD_URL_FORMAT, www, wikiaName, envType.getWikiaDomain());
       }
       case STAGING: {
-        return String.format(PROD_URL_FORMAT, www, wikiaName, env.getWikiaDomain());
+        return String.format(PROD_URL_FORMAT, www, wikiaName, envType.getWikiaDomain());
       }
       case SANDBOX: {
-        return String.format(SANDBOX_URL_FORMAT, this.env, www, wikiaName, env.getWikiaDomain());
+        return String.format(SANDBOX_URL_FORMAT, this.env, www, wikiaName, envType.getWikiaDomain());
       }
       default:
         return "";
     }
   }
 
+  public String getWikiGlobalURL(){
+    EnvType env = Configuration.getEnvType(this.env);
+
+    switch (env) {
+      case DEV: {
+        String devBoxOwner = this.env.split("-")[1];
+        return String.format("http://%s.www.%s", devBoxOwner, env.getWikiaDomain());
+      }
+      case SANDBOX: {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Extract this nested code block into a method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1199)